### PR TITLE
Minor video streaming phrasing updates

### PR DESCRIPTION
**Pull request by lx-is**
changes: https://github.com/privacyguides/privacyguides.org/pull/1077/files#diff-8adec93397671c861254b923908e7a16e1cd9091e44a8f2c73b80140d806fd3e

### CODE REVIEW

File: docs/video-streaming.en.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ The primary threat when using a video streaming platform is that your streaming
 
 !!! Warning
 
-    When using FreeTube, your IP address is still known to YouTube, [Invidious](https://instances.invidious.io) and the SponsorBlock instances that you use. Consider using a [VPN](vpn.md) or [Tor](https://www.torproject.org) if your [threat model](threat-modeling.md) requires hiding your IP address.
+    When using FreeTube, your IP address is still known to YouTube & the [Invidious](https://instances.invidious.io) and SponsorBlock instances that you may use. Consider using a [VPN](vpn.md) or [Tor](https://www.torproject.org) if your [threat model](threat-modeling.md) requires hiding your IP address.
```
Comment (jonaharagon) [inline docs/video-streaming.en.md:13]: I'm not sure if this change actually makes more sense.

File: docs/video-streaming.en.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The primary threat when using a video streaming platform is that your streaming
 
 !!! note
 
-    Only the **LBRY desktop client** is recommended. The [Odysee](https://odysee.com) website and the LBRY clients in F-Droid, Play Store, and the AppStore have mandatory synchronization and telemetry.
+    Only the **LBRY desktop client** is recommended as the [Odysee](https://odysee.com) website and the LBRY clients in F-Droid, Play Store, and the App Store have mandatory synchronization and telemetry.
```
Comment (jonaharagon) [inline docs/video-streaming.en.md:36]:
```suggestion
    Only the **LBRY desktop client** is recommended, as the [Odysee](https://odysee.com) website and the LBRY clients in F-Droid, Play Store, and the App Store have mandatory synchronization and telemetry.
```

File: docs/video-streaming.en.md (hunk 1 above)
Comment (jonaharagon) [reply docs/video-streaming.en.md:13]:
```suggestion
    When using FreeTube, your IP address is still known to YouTube, and the [Invidious](https://instances.invidious.io) or SponsorBlock instances that you may use. Consider using a [VPN](vpn.md) or [Tor](https://www.torproject.org) if your [threat model](threat-modeling.md) requires hiding your IP address.
```

Comment (lx-is) [reply docs/video-streaming.en.md:13]: > Edit: see your own suggestion! https://github.com/github/explore/blob/main/docs/styleguide.md#ampersands

aaa lol

I think this makes the most sense:

```suggestion
    When using FreeTube, your IP address may still be known to YouTube, [Invidious](https://instances.invidious.io), or [SponsorBlock](https://sponsor.ajay.app/) depending on your configuration. Consider using a [VPN](vpn.md) or [Tor](https://www.torproject.org) if your [threat model](threat-modeling.md) requires hiding your IP address.
```

**Review by jonaharagon — APPROVED** [review]

**Pull request merged**